### PR TITLE
Add method for creating file

### PR DESCRIPTION
**Pull request by akharche**
It is needed for configuration, where we should create and pack addition file with environment settings.

### CODE REVIEW

File: common/helper.py [hunk 1]
```diff
@@ -660,3 +660,17 @@ def get_packing_cmd(pack_type, pack_dir, enable_ruby, version, source_name):
...
+    Create file and write data
+
+    :param file_path: path to file
+    :type: String
+
+    :param data: data to write
+    :type: String
+    """
+
+    pathlib.Path(file_path).write_text(data, encoding='utf-8')
```
Comment (azhogov) [inline common/helper.py]: what about error handling?

**Comment by azhogov** [comment]
> Can this function be called directly from product_config yeah?
According to this review the function is a part of product configs API

**Comment by akharche** [comment]
@GoAlexander Possibly we need to call this function in several configs, so its better to make it a part of API

File: common/helper.py [hunk 2]
```diff
@@ -660,3 +660,25 @@ def get_packing_cmd(pack_type, pack_dir, enable_ruby, version, source_name):
...
+    """
+    log = logging.getLogger('helper.create_file')
+
+    log.info('-' * 50)
+    log.info(f'create file {file_path}')
+
+    try:
+        pathlib.Path(file_path).write_text(data, encoding='utf-8')
+    except Exception as error:
+        log.error(f"Can not create file {file_path}: {error}")
```
Comment (azhogov) [inline common/helper.py]: what about return status?

File: common/helper.py [hunk 3]
```diff
@@ -660,3 +660,25 @@ def get_packing_cmd(pack_type, pack_dir, enable_ruby, version, source_name):
...
+    """
+    Create file and write data
+
+    :param file_path: path to file
+    :type: String
+
+    :param data: data to write
+    :type: String
+    """
+    log = logging.getLogger('helper.create_file')
```
Comment (azhogov) [inline common/helper.py]: Please check coexistence with the Build Scrips logging

**Comment by GoAlexander** [comment]
> @GoAlexander Possibly we need to call this function in several configs, so its better to make it a part of API

Great, it is 100% what wanted.

File: common/helper.py [hunk 4]
```diff
@@ -663,3 +663,21 @@ def get_packing_cmd(pack_type, pack_dir, enable_ruby, version, source_name):
...
+    :param file_path: path to file
+    :type: String
+
+    :param data: data to write
+    :type: String
+    """
+
+    try:
+        pathlib.Path(file_path).write_text(data, encoding='utf-8')
+    except Exception:
```
Comment (azhogov) [inline common/helper.py]: except is not needed in this case

**Pull request merged**